minorminer/layout/layout.py

```diff
@@ -19,11 +31,147 @@ def kamada_kawai(G, d=2, **kwargs):
     layout : dict
         A mapping from vertices of G (keys) to points in R^d (values).
     """
+    # Set center to a default value based on d.
+    center = center or d*(0, )
+    center = np.array(center)
+
     # NetworkX has a bug #3658.
     # Once fixed, these can collapse and `dim=n` can be part of `**kwargs`.
     if d in (1, 2):
-        return nx.kamada_kawai_layout(G, dim=d, **kwargs)
+        return nx.kamada_kawai_layout(G, dim=d, center=center, scale=scale, **kwargs)
     else:
+        # The random_layout is in [0, 1]^d
+        random_layout = nx.random_layout(G, dim=d, seed=seed)
+
+        # Convert it to [center - scale, center + scale]^d
+        transformed_random_layout = scale_unit_layout(
+            random_layout, center, scale)
+
         return nx.kamada_kawai_layout(
-            G, center=d*(1/2, ), scale=1/2, pos=nx.random_layout(G, dim=d), dim=d, **kwargs
-        )
+            G, pos=transformed_random_layout, dim=d, center=center, scale=scale, **kwargs)
+
+
+def chimera(G, d=2, center=None, scale=1., **kwargs):
```

Comment:
is a similar function for pegasus coming down the line?

Reply:
I have a `dnx_layout` coming down the line--handles both architectures.